src/app/resourcebundles/data/consumption/en.properties

```diff
@@ -879,6 +879,7 @@ messages.stmsg.m0140 = DOWNLOADING
 messages.stmsg.m0141 = Data unavailable to generate Score Report
 messages.stmsg.m0142 = PAUSED
 messages.stmsg.m0143 = DOWNLOAD
+messages.stmsg.m0144 = No Content Available
 orgname = orgname
 participants = Participants
```

Comment:
add en.json file as well. These messages needs to be added in phaser app as well.